cherryontech/jupiter-jumpers
# Adds initial landing page template

## Description

1) Adds the initial template for the landing page, including parallax background and fixed navbar.

## Related Story Card

[SCRUM-51](https://cherryontech-jupiter-jumpers.atlassian.net/browse/SCRUM-51)

## Type of Changes

New page

## Acceptance Criteria

- [X] Given I am visiting the website and am not logged in,
- [X] When I navigate to the platform's main URL,
- [X] Then the system displays Uncloud’s landing page that showcases the platform's core functionalities and benefits, and invites the user to “Start Now” (makes account creation clear).

## Update Screenshots

### After
![Screenshot 2024-06-09 at 3 45 13 PM](https://github.com/cherryontech/Uncloud/assets/47403135/6cf625cb-15f9-4364-b767-bb5e7b4d296c)

## Testing Instructions

1. Play around with different window sizes, and try scrolling; ensure that the background looks okay (there should be a cloudy background at the top, and it should transition to a white background).
2. Ensure the links to log in and register work, and that those pages look okay with the updated navbar.
